src/Parser/PhpDocParser.php

```diff
@@ -346,6 +346,14 @@ private function parseMethodTagValue(TokenIterator $tokens): Ast\PhpDoc\MethodTa
 			exit;
 		}
 
+		$templateTypes = [];
+		if ($tokens->tryConsumeTokenType(Lexer::TOKEN_OPEN_ANGLE_BRACKET)) {
+			do {
+				$templateTypes[] = $this->parseMethodTagValueTemplateType($tokens);
```

Comment:
Could we call the already existing `parseTemplateTagValue`?

Reply:
@ondrejmirtes 
I fixed it.
`parseTemplateTagValue` also parse `description` text, so it was necessary to add the flag parameter  `$parseDescription`.